src/components/upload-file/index.module.scss

```diff
@@ -1,82 +1,93 @@
 @import "../../variables";
 
-.wrapContent {
-  margin-top: 5vh;
-    display: flex;
-    justify-content: center;
-    .square {
-        height: 510px;
-        width: 566px;
-        background-color: #EDEDED;
-        margin-right: 50px;
-    }
-    
-    .rightContent {
-      margin-top: 5vh;
-      display: flex;
-      flex-direction: column;
-
-      label {
-        color: #009473;
-        margin-bottom: 10px;
-        font-size: 14px;
-      }
-
-      .boxInput {
-        border: 0.5px solid #009473;
-        border-radius: 2px;
-        display: inline;
-        padding: 0 20px;
-        width: 155px;
-        margin-bottom: 30px;
-        i {
-          font-size: 18px;
-        }
-        select {
-          padding: 5px 5px;
-          border: none;
-          font-size: 14px;
+@media only screen and (max-width: 810px) {
+  body {
+      .wrapContent {
+        .square {
+          height: 410px;
+          width: 466px;
         }
       }
+  }
+}
 
-      .buttonUploadFile {
-        color: #000000;
-        border: 0.5px solid #009473;
-        border-radius: 2px;
-        padding: 5px 20px;
-        margin-bottom: 4vh;
-        svg {
-          width: 18px;
-          height: 18px;
-          margin-right: 10px;
+
+@media only screen and (max-width: 650px) {
+  body {
```

Comment:
dangerous dude, but ok if you checked everything

Reply:
Edit Done